AutoResizeFont/ViewController.swift

```diff
@@ -9,7 +9,9 @@
 import UIKit
 
 class ViewController: UIViewController {
-
+    
+    @IBOutlet var label1 : DynamicLabel!
```

Comment:
This outlet is not necessary.

Reply:
I know, I only added it as a test. there are two labels but I only created one outlet for testing purposes.

Reply:
it does not have to be in the production code, will remove it as directed